__tests__/integration/repeatableResources.test.js

```diff
@@ -36,7 +36,7 @@ describe('Adding new embedded Resource Templates', () => {
   it('clicks on a nested property with an embedded Note resource template and then clicks on the AddButton for a second resource template', async () => {
     await pupExpect(page).toClick('a[data-id=\'note\']')
     await pupExpect(page).toClick('.col-sm-4 > div:nth-child(1) > button:nth-child(1)')
```

Comment:
might it be possible to have a better selector for the add button that is more obviously ... the add button for the template?  I think we should avoid "nth-child" and classes that are rendering instructions when possible ...